### PR TITLE
test(integration): setup

### PR DESCRIPTION
**Pull request by byeongsu-hong**
resolves MITO-18
resolves MITO-110

* nodes
    * osmo -> test-tube
    * evm -> anvil
* test contracts
    * test-mock-ism
    * test-mock-msg-receiver
* bech32
    * applied to parse logic in Message receiver
    * `hpl-interface/types/lib.rs`
* testing
    * evm_mailbox <-> cw_mailbox
    * TBD...

### CODE REVIEW

File: integration-test/tests/setup/mod.rs [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+}
+
+#[tokio::test]
+async fn test_setup() {
+    let test_env = setup_env().await.unwrap();
+
+    println!("{:?}", test_env.cw_deployments.addrs);
+    println!("{:?}", test_env.cw_deployments.codes);
+    println!("{:?}", test_env.evm_deployments);
+}
```
Comment (hashableric) [inline integration-test/tests/setup/mod.rs]: please remove stdio macros in test

**Review by hashableric — COMMENTED** [review]
please write some comment on testing codes. hard to figure out whats your purpose is. @byeongsu-hong

**Review by hashableric — CHANGES_REQUESTED** [review]
and please make ci pass. @byeongsu-hong

File: integration-test/tests/setup/mod.rs (hunk 1 above)
Comment (byeongsu-hong) [reply integration-test/tests/setup/mod.rs]: just added this test to check the address of deployments

**Review by steve806 — APPROVED** [review]
Looks Good To Me

**Pull request merged**